src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -896,4 +896,75 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>Package content hash validation failed for {0}. Expected: {1} Actual: {2}</value>
     <comment>0 - package ID, 1 - expected Sha, 2 - actual sha</comment>
   </data>
+  <data name="Error_SourceProviderIsNull" xml:space="preserve">
+    <value>Property SourceProvider is null.</value>
+  </data>
+  <data name="SourcesCommandNameRequired" xml:space="preserve">
+    <value>The name specified cannot be empty. Please provide a valid name.</value>
+  </data>
+  <data name="SourcesCommandNoMatchingSourcesFound" xml:space="preserve">
+    <value>Unable to find any package source(s) matching name: {0}.</value>
+  </data>
+  <data name="SourcesCommandSourceDisabledSuccessfully" xml:space="preserve">
+    <value>Package source with Name: {0} disabled successfully.</value>
+  </data>
+  <data name="SourcesCommandSourceEnabledSuccessfully" xml:space="preserve">
+    <value>Package source with Name: {0} enabled successfully.</value>
+  </data>
+  <data name="SourcesCommandSourceRemovedSuccessfully" xml:space="preserve">
+    <value>Package source with Name: {0} removed successfully.</value>
+  </data>
+  <data name="ReservedPackageNameAll" xml:space="preserve">
+    <value>All</value>
+  </data>
+  <data name="SourcesCommandAllNameIsReserved" xml:space="preserve">
+    <value>Package source name 'All' is a reserved name.</value>
+  </data>
+  <data name="SourcesCommandSourceRequired" xml:space="preserve">
+    <value>The source specified cannot be empty. Please provide a valid source.</value>
+  </data>
+  <data name="SourcesCommandInvalidSource" xml:space="preserve">
+    <value>The source specified is invalid. Please provide a valid source.</value>
+  </data>
+  <data name="SourcesCommandUniqueName" xml:space="preserve">
+    <value>The name specified has already been added to the list of available package sources. Please provide a unique name.</value>
```

Comment:
Should we use 'please' in the text messages?

Reply:
I was just copying strings from nuget.commandline -- but yes, good catch. will stop being overly nice. :-)

Reply:
i fixed all cases of "please" inside this file. (i left please in localization comments).